compose/docker-compose.yml

```diff
@@ -151,6 +151,9 @@ services:
         - GALAXY_CONFIG_AMQP_INTERNAL_CONNECTION=amqp://galaxy:vaiJa3ieghai2ief0jao@rabbitmq/galaxy 
         # Condor
         - GALAXY_DESTINATIONS_DEFAULT=${GALAXY_DESTINATIONS_DEFAULT:-slurm_cluster}
+        - GALAXY_RUNNERS_ENABLE_CONDOR=${GALAXY_RUNNERS_ENABLE_CONDOR:-False}
```

Comment:
Afaik, the .env file is just used to replace envs in the docker-compose.yml file. It will not passed along. So this looks better, you could even leave the lines in the different env files.